config.json

```diff
@@ -18,6 +18,7 @@
       "storeTransactions",
       "getTrytes"
     ],
+    "whitelistedAddresses": [],
```

Comment:
Afaik, current behavior is to whitelist `::1` and `127.0.0.1` by default, maybe this should be specified here by default, keeps it more visible, avoid confusion?